src/main/java/org/zone/commands/structure/misc/Messages.java

```diff
@@ -0,0 +1,307 @@
+package org.zone.commands.structure.misc;
+
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+import org.spongepowered.api.entity.living.player.User;
+import org.spongepowered.configurate.ConfigurateException;
+import org.zone.ZonePlugin;
+import org.zone.commands.system.ArgumentCommand;
+import org.zone.commands.system.context.ErrorContext;
+import org.zone.region.Zone;
+import org.zone.region.flag.FlagType;
+import org.zone.region.flag.move.player.leaving.LeavingFlag;
+import org.zone.region.group.Group;
+
+import java.math.BigDecimal;
+import java.util.Collection;
+import java.util.UUID;
+
+/**
+ * Store house for all messages in this plugin {@link ZonePlugin} to make translation easier
+ */
+public final class Messages {
+
+    private Messages() {
+        throw new RuntimeException("Could not construct class");
+    }
+    //Universal Messages
+    public static Component setUniversalPlayerOnlyCommandErrorMessage() {
```

Comment:
I would rename this to ``getPlayerOnlyErrorMessage()`` or something like that. 
Essentially, keep the messages simple and assume that all of them are universal, even if they get called the once

Reply:
ok